### PR TITLE
start work on coworking page

### LINKED ISSUE #591: add coworking page
cc @steffilazerte @yabellini

### PR DESCRIPTION
**Pull request by maelle**
Fix #591 

Changes necessary in future coworking events Markdown files

- Add "coworking: true"
- Make the YAML description field a bit more specific as it's used on future event cards.

@steffilazerte on the comm calls page, there's a way to suggest future ideas. How could it happen for coworking? Could people suggest repeat sessions?

### CODE REVIEW

**Comment by steffilazerte** [comment]
Looks soooo good!

**Comment by steffilazerte** [comment]
Questions
- Will it show all upcoming sessions or just the next one? If only the next one, perhaps add a link to the events page à la "See Events for future sessions and other fun things rOpenSci is doing"
- Where should I put links to the coworking documents? I could add them to the event listing after the session? It'd be nice for there to be a firm home for them...

**Comment by maelle** [comment]
> Will it show all upcoming sessions or just the next one? If only the next one, perhaps add a link to the events page à la "See Events for future sessions and other fun things rOpenSci is doing"

All upcoming sessions!

> Where should I put links to the coworking documents? I could add them to the event listing after the session? It'd be nice for there to be a firm home for them...

I've now added them to previous events (btw the July one should become read only) with a `notes` YAML field, do you like how they appear on the listing of past events and on the past events' single pages?

**Comment by maelle** [comment]
@steffilazerte now you can see several sessions

image

**Comment by maelle** [comment]
@steffilazerte could you please add some more text in `content/coworking/_index.md`? This way it'd look better with the image.

I can tackle further comments next week.

**Comment by maelle** [comment]
The text in `content/coworking/_index.md` could answer this question

> @steffilazerte on the comm calls page, there's a way to suggest future ideas. How could it happen for coworking? Could people suggest repeat sessions?

File: content/events/2021-09-07-coworking-2021-09.md
Comment (steffilazerte) [inline content/events/2021-09-07-coworking-2021-09.md:1]: Good to remember!

**Comment by steffilazerte** [comment]
- Why do some events say "online" and some say "teleconference"? https://deploy-preview-602--ropensci.netlify.app/events/ I don't feel too strongly (other than a desire for consistency), but my preference would be for 'online' (a bit more casual sounding)
- I like the placement of "Collaborative notes" on the listings page. On the events page is not bad, but I'm not sure it's very obvious. Would it be better at the bottom under a "Resources" heading? 
  - Pro: Same set up as Community Calls (cf Community calls https://ropensci.org/commcalls/jun2023-arrow/)
  - Con: There is only one resource, so not sure it really makes sense to have a whole heading just for that!

**Comment by yabellini** [comment]
I like how looks.

**Comment by maelle** [comment]
@steffilazerte regarding the text, it's great! I changed RSVP to registration because I started thinking it might be clearer to non native speakers. :thinking: Could you clarify how to contact you or Yani? For instance, can people email you? When you say on Mastodon, do you mean tagging the rOpenSci account or another one?

**Comment by maelle** [comment]
> Why do some events say "online" and some say "teleconference"? [deploy-preview-602--ropensci.netlify.app/events](https://deploy-preview-602--ropensci.netlify.app/events/) I don't feel too strongly (other than a desire for consistency), but my preference would be for 'online' (a bit more casual sounding)

changed!

> I like the placement of "Collaborative notes" on the listings page. On the events page is not bad, but I'm not sure it's very obvious. Would it be better at the bottom under a "Resources" heading?

I made it bolder, because I fear that one could miss the link if it were below a resources heading since one would need to scroll down?

Do you think one could, in the future, remove the first sentence of each comm call content since it's redundant with the information that's above and makes it all look a bit busy? The one starting with "join us". On the other hand I can see it shows another timezone

image

**Comment by maelle** [comment]
@steffilazerte once you've read/tackled my last two comments, feel free to squash and merge if the build is green! Unless you disagree with the collaborative notes link. :smile_cat:

**Comment by steffilazerte** [comment]
Hi @maelle 
- I like the bolder text, thank you!
- I clarified how to contact Yani and I. For mastodon, it doesn't matter how they reach out (my mastodon is pretty much just for rOpenSci). @yabellini is that okay for you?
- I think I'll leave the intro sentence for now, the timezone is an important thing to have as it's the 'anchor' point (as daylight savings changes, the UTC time changes), and I like that people nearby can see at a glance what time it would be for them without having to look it up. But I'll think about how to make that less repetitive...
- The preview still says "teleconference" do you know why?

**Comment by yabellini** [comment]
I’m ok with the way to contact me.

**Pull request merged**